pkgs/development/python-modules/iterfzf/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fzf, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "iterfzf";
+  version = "0.5.0.20.0";
+
+  src = fetchFromGitHub {
+    owner = "dahlia";
+    repo = pname;
+    rev = version;
+    sha256 = "0nrfq1zj4pwlgyd8ra5wl7skdbzvnxdh7wwhyh83y7yax8sp1xvj";
+  };
+
+  prePatch = ''
+    substituteInPlace iterfzf/__init__.py \
+                      --replace "EXECUTABLE_NAME = 'fzf.exe' if sys.platform == 'win32' else 'fzf'" \
+                      "EXECUTABLE_NAME = 'fzf.exe' if sys.platform == 'win32' else '${fzf}/bin/fzf'"
+  '';
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Reply:
It looks like the sources have no tests. So, I'd say `pythonImportsCheck` is the best we can do right now.